### DISCUSSION TITLE
Login Failed

### DISCUSSION THREAD

**Discussion by gfo2007** [abpframework/abp#15852] [Q&A] [answered]
Hi,
I have just created a new abp project. When I try to login in, it gives me a login failed error even though I am using the default username and password to login. Image is below.

Username: admin
Password: 1q2w3E*

![Login Failed](https://user-images.githubusercontent.com/14317420/222349669-d42c80dd-43f9-4458-9e18-52052cc0e59f.PNG)

Please help.

**Comment by gfo2007** [comment]
There was no .DBMigrator in the Single Layer Application to seed the admin user data.

**Comment by berkansasmaz** [comment, marked as answer]
See: https://docs.abp.io/en/abp/latest/Startup-Templates/Application-Single-Layer#how-to-run

**Reply by gfo2007** [reply]
Thank you.
